libs/langchain/langchain/chat_models/vertexai.py

```diff
@@ -132,17 +132,19 @@ def is_lc_serializable(self) -> bool:
     def validate_environment(cls, values: Dict) -> Dict:
         """Validate that the python package exists in environment."""
         cls._try_init_vertexai(values)
+        preview = values["preview"]
```

Comment:
My comment is not about this PR change so, ignore it if you like.
The `values` semantic is the `kwargs` semantic. I'd rename `values` to `kwargs`.

Reply:
noted, thanks, will change in the enxt iterations then!